source/open-telemetry/tests/operation/insert.yml

```diff
@@ -0,0 +1,63 @@
+description: operation insert
+schemaVersion: '1.26'
+createEntities:
+  - client:
+      id: client0
+      useMultipleMongoses: false
+      observeTracingMessages:
+        enableCommandPayload: true
+  - database:
+      id: database0
+      client: client0
+      databaseName: operation-insert
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: test
+initialData:
+  - collectionName: test
+    databaseName: operation-insert
+    documents: []
+tests:
+  - description: insert one element
+    operations:
+      - object: collection0
+        name: insertOne
+        arguments:
+          document:
+            _id: 1
+    expectTracingMessages:
+      client: client0
+      ignoreExtraSpans: false
+      spans:
+        - name: insert operation-insert.test
+          tags:
+            db.system: mongodb
+            db.namespace: operation-insert
+            db.collection.name: test
+            db.operation.name: insert
+            db.operation.summary: insert operation-insert.test
+          nested:
+            - name: command insert
+              tags:
+                db.system: mongodb
+                db.namespace: operation-insert
+                server.address: { $$type: string }
+                server.port: { $$type: [ 'long', 'string' ] }
+                server.type: { $$type: string }
+                db.query.summary: insert
+                db.query.text:
+                  $$matchAsDocument:
+                    $$matchAsRoot:
+                      insert: test
+                      ordered: true
+                      txnNumber: { $$unsetOrMatches: 1 }
```

Comment:
Why would this use `$$unsetOrMatches`? I don't see that in any existing spec tests. Most assert that the value is `1` (or use `$numberLong`, which is redundant due to numeric comparison rules in the test runner).